starport/services/scaffolder/scaffolder.go

```diff
@@ -60,15 +65,42 @@ func AddressPrefix(prefix string) Option {
 	}
 }
 
+// NewChain initializes a new chain Scaffolder at path.
+func NewChain(path string, options ...Option) (*Scaffolder, error) {
```

Comment:
Can't see why we need NewChain specially when it's still possible to call Init func when you initialize scaffolder with New. This is ambiguous in terms of API design, I think we should just have a New.